docs/install/keycloak.md

```
@@ -0,0 +1,176 @@
+helm upgrade dave ./dave -f values.yaml -f values-${ENVIRONMENT}.yaml -f my-values-keycloak.yaml --install
+```
+
+5. Verify pods and login flow.
+
+---
+
+## Keycloak realm & client setup
+
+Follow these steps to prepare Keycloak to work with DAVe:
+- Create a realm (example: `dave`). 
+- Import the client configuration from [here](https://github.com/it-at-m/dave-backend/blob/sprint/sso-config/sso-client.json).
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)